### DISCUSSION TITLE
Global bibliography

### DISCUSSION THREAD

**Discussion by cranmer** [executablebooks/meta#110] [Q&A]
I see that 
````
```{bibliography} references.bib
```
````
yields a bibliography with the references mentioned with `{cite}` on that page, but I wanted to create a bibliography for the entire book. Is that possible?

**Comment by chrisjsewell** [comment]
Heya, I can't remember of the top of my head, you might want to have a look at https://sphinxcontrib-bibtex.readthedocs.io/en/latest/usage.html and see if anything works for you?

But I think it makes sense to have this option available in the jupyter-book _config.yml

**Reply by cranmer** [reply]
Maybe the issue is that I had an extra cite in a page that came after the bibliography. Some discussion of issue and workarounds here;
https://sphinxcontrib-bibtex.readthedocs.io/en/latest/usage.html#issue-unresolved-citations

**Reply by choldgraf** [reply]
Yeah we really need to document this workaround a bit better, but in the long term need to figure out how to upstream an improvement to that package so that this is no longer an issue, because I think folks will find it quite unintuitive if their bibliography behavior changes according to the file names :-/

**Reply by chrisjsewell** [reply]
agree, I've contributed to sphinxcontrib-bibtex before, the maintainer is amenable
